kernel/kernel_align_generic.c

```diff
@@ -33,10 +33,10 @@
 *                                                                                                 *
 **************************************************************************************************/
 
-
+#include <stdint.h>
 
 void blasfeo_align_64_byte(void *ptr, void **ptr_align)
 	{
-	*ptr_align = (void *) ( ( ( (unsigned long long) ptr ) + 63) / 64 * 64 );
+	*ptr_align = (void *) ( ( ( (uintptr_t) ptr ) + 63) / 64 * 64 );
```

Comment:
`uintptr_t` seems to be defined as an optional type in C99. So it would be good to have a check if C99 is used (note to myself to fix later).

Reply:
Interesting. I was not aware that they are optional. I have had some problems with other optional parts of C99 in the past but these seem to be well supported in my experience.

Reply:
I got they are optional from a web search. Anyway, glad to hear that they are usually well supported. I will still add a check for C99, since BLASFEO should (in theory) build also in systems where C99 it is not supported.